uw-frame-components/portal/features/controllers.js

```diff
@@ -53,10 +53,10 @@ define(['angular','require'], function(angular, require) {
 
 
      //local functions ---------------------------------------------------------
-     
+
      var getPopups = function(){
        portalFeaturesService.getUnseenPopups().then(function(unSeenPopups) {
-         if(unSeenPopups.length !=0){
+          if(unSeenPopups.length !=0 && !$rootScope.GuestMode){
```

Comment:
Should we even call the `portalFeaturesService.getUnseenPopups()` if you are a guest? I would think this check is before this return from a service call.